.github/workflows/label-actions.yml

```diff
@@ -4,6 +4,9 @@ on:
   issues:
     types: [labeled]
 
+permissions:
```

Comment:
The example config from `dessant/label-actions` in their README gives these permissions.
See https://github.com/dessant/label-actions#examples

```yml
name: 'Label Actions'

on:
  issues:
    types: [labeled, unlabeled]
  pull_request:
    types: [labeled, unlabeled]

permissions:
  contents: read
  issues: write
  pull-requests: write

jobs:
  action:
    runs-on: ubuntu-latest
    steps:
      - uses: dessant/label-actions@v2
 ```

We don't process pull requests so we should NOT give it any `pull-requests` permission. 😄 

But maybe the action does need the `contents: read` permission, not sure about that.

https://docs.github.com/en/rest/reference/permissions-required-for-github-apps#permission-on-contents